app/scripts/util/tip.js

```diff
@@ -11,6 +11,8 @@ const Tip = function(el, config) {
     this.hideTimeout = null;
     this.force = config && config.force || false;
     this.hide = this.hide.bind(this);
+    this.mouseenterHandler = null;
```

Comment:
Why do we need all this mouse magic instead of using the standard tooltips?

Reply:
This was the system in place that's used for tge refresh button. I think we don't use native tooltips because you can't style them. Maybe it also has something to do with positioning.

Reply:
I mean, not the OS tooltip, but the one KeeWeb has: https://github.com/keeweb/keeweb/blob/master/app/templates/generator.hbs#L3. When you add `title` attribute, it will render a custom tooltip. What's the need of adding this additional logic on top of it?

Reply:
The custom tooltips are created from the title attributes of a jquery element `e` and all it's children when `Tip.createTips(e)` is called. Unfortunately, it's only called once, but the tooltip needs to be recreated each time the toggle button is pressed, since the tooltip changes back and forth from "Hide password" to "Show password". Setting the title attribute doesn't automatically regenerate the tooltip.

Reply:
I just realized that you're saying you'd prefer a non-changing tooltip that says "Hide/show password" to reduce code complexity. Is that correct?